app.py

```diff
@@ -122,21 +123,31 @@ def pem_create():
     else:
         return url_for('pem_router')
 
-@app.route("/config/pem/form/delete", methods=["POST"])
+@app.route("/config/pem/form/delete", methods=["GET"])
```

Comment:
delete method should be `delete`. check [this](https://pythonise.com/series/learning-flask/flask-http-methods#delete-requests) out.

Reply:
I'll check it